### PR TITLE
Re-skin sparklines

### PR DESCRIPTION
**Pull request by westlywright**

### CODE REVIEW

File: app/components/container-row/template.hbs [hunk 1]
```diff
@@ -25,10 +25,10 @@
 {{#if showStats}}
   <td class="spark-td">
     {{#if model.cpuSpark}}
-      {{spark-line data=model.cpuSpark     width=60 height=15 max=cpuMax     prefix="CPU: " formatter="percent"}}
-      {{spark-line data=model.memorySpark  width=60 height=15 max=memoryMax  prefix="Memory: " formatter="mib"}}
-      {{spark-line data=model.networkSpark width=60 height=15 max=networkMax prefix="Network: " formatter="kbps"}}
-      {{spark-line data=model.storageSpark width=60 height=15 max=storageMax prefix="Storage: " formatter="kbps"}}
+      {{spark-line data=model.cpuSpark     width=80 height=30 max=cpuMax     prefix="CPU: " formatter="percent" type="cpu"}}
```
Comment (vincent99) [inline app/components/container-row/template.hbs]: Don't these look weird at 80px of chart and 60 points = 1.33px per point on a non-retina monitor?

File: app/components/spark-line/component.js [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
   classNames: ['spark-line'],
-  attributeBindings: ['cssSize:style','tooltip'],
+  attributeBindings: ['cssSize:style', 'tooltip'],
 
   width: null,
   height: 20,
   min: 0,
   max: null,
-  interpolation: 'step-after',
+  interpolation: 'linear',
```
Comment (vincent99) [inline app/components/spark-line/component.js]: Hmm, I liked the step because it makes the area filled accurate...

Comment (westlywright) [inline app/components/spark-line/component.js]: I can change this back, i had changed it to match the mocks. The bigger charts are also linear in the mocks do we want to keep them stepped?

File: app/components/container-row/template.hbs (hunk 1 above)
Comment (westlywright) [inline app/components/container-row/template.hbs]: I used RDM to change my screen res to 1400x900 non-HiDPI mode. They look a little pixelated. I'll change it back.

**Comment by westlywright** [comment]
@vincent99 updated per comments

**Pull request merged**